modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc

```diff
@@ -54,6 +54,6 @@ $ virt-install --noautoconsole \
    --disk {disk} \
    --import \
    --network network={network},mac={mac} \
-   --qemu-commandline="-drive \
+   --disk="-drive \
 if=none,id=ignition,format=raw,file={ign_file},readonly=on -device virtio-blk,serial=ignition,drive=ignition"    
```

Comment:
The syntax of the parameters after ``--disk`` is different:
```
--disk path={ign_file},format=raw,readonly=on,serial=ignition,startup_policy=optional
```

`-device virtio-blk,serial=ignition,drive=ignition` can be dropped.

Reply:
@cfergeau Addressed- Does it look good?

Reply:
This really should be `--disk path={ign_file},format=raw,readonly=on,serial=ignition,startup_policy=optional`, not 
```
   --disk path={ign_file} \
if=none,id=ignition,format=raw,file={ign_file},readonly=on,startup_policy=optional"
```

Reply:
Addressed